plugins/newsdownloader.koplugin/main.lua

```diff
@@ -388,6 +385,54 @@ function NewsDownloader:setCustomDownloadDirectory()
     }:chooseDir()
 end
 
+function NewsDownloader:changeFeedConfig() 
+    local feed_config_file = io.open(feed_config_path, "rb")
+    local config = feed_config_file:read("*all")
+    feed_config_file:close()
+    config_editor = InputDialog:new{ --todo change to local var, requires fix in save and cancel. No idea how.
+        title = _("Feed config editor"),
+        input = config,
+        input_type = "string",
+        fullscreen = true,
+        condensed = true,
+        allow_newline = true,
+        cursor_at_end = false,
+        add_scroll_buttons = true,
+        add_nav_bar = true,
```

Comment:
You don't need both, choose add_scroll_buttons OR add_nav_bar .

Reply:
ok. thanks